docs/Developers/Newcomers_Guide.rst

```diff
@@ -126,22 +126,14 @@ Now you are ready to join coala community at `coala gitter <https://coala.io/cha
 The newcomers should ping us "Hello World" to let us know they are here
 because we care!
 
-**Congratulations!** You are now part of our community.
-
-Step 2. Grab an Invitation to the Organization
-----------------------------------------------
-
-Let us know on gitter that you are interested in contributing and ask for an
-invitation to our org. This is the first step towards contributing.
-A maintainer will command ``cobot`` (our gitter bot) to invite
-you to the Newcomer team.
-The invitation will be sent by email and you will have to accept
-it to join. If you don't see the invitation, accept it on our `team page
-<https://github.com/coala>`__.
+When you say "Hello World" in chat ``cobot`` (our gitter bot) will invite you
```

Comment:
do i create a new commit for changing all mentions of cobot to corobo or include it in one of the already existing ones?